beprof/profile.py

```diff
@@ -1,5 +1,6 @@
 from beprof.curve import Curve
 import numpy as np
+from beprof import functions as f
```

Comment:
I see some contraction in importing Curve and functions.
First one you do using `from x.y import z` and the second one you do using `from x import y as z`. Why ?